pkg/config/manager_remote.go

```diff
@@ -133,7 +133,41 @@ func (m *RemotePersistenceManager) PersistSourceBindings(ctx context.Context, co
 }
 
 func (m *RemotePersistenceManager) PersistActionEnabled(ctx context.Context, name string, enabled bool) error {
-	return errors.New("PersistActionEnabled is not implemented for GQL manager")
+	logger := m.log.WithFields(logrus.Fields{
+		"deploymentID":    m.gql.DeploymentID(),
+		"resourceVersion": m.getResourceVersion(),
+		"actionName":      name,
+		"enabled":         enabled,
+	})
+	logger.Debug("Updating action configuration")
+
+	err := m.withRetry(ctx, logger, func() error {
+		var mutation struct {
+			Success bool `graphql:"patchDeploymentConfig(id: $id, input: $input)"`
+		}
+		variables := map[string]interface{}{
+			"id": graphql.ID(m.gql.DeploymentID()),
+			"input": remoteapi.PatchDeploymentConfigInput{
+				ResourceVersion: m.getResourceVersion(),
+				Action: &remoteapi.ActionPatchDeploymentConfigInput{
+					Name:    name,
+					Enabled: &enabled,
```

Comment:
here we ahve `enabled` where for notif we have `disabled`:
```go
// ...
ChannelAlias:           channelAlias,
Disabled:               !enabled,
```

"the consistency is strong with this one" 😅